Create an ADOPTERS page (#504)
cnoe-io/idpbuilder: ADOPTERS.md

ADOPTERS
========

PUBLIC REFERENCES
-----------------

> This document lists the organizations that use Idpbuilder based on public information available in blog posts and videos.

Reviewer: ```suggestion
This document lists the organizations that use Idpbuilder
```
We want to encourage any org that uses idpbuilder tool to submit a PR. They might no have public info mentioning. But is encourage. 
Here is an example on ArgoCD list adopters/users https://github.com/argoproj/argo-cd/blob/master/USERS.md

> ADOPTERS
> ========
> 
> PUBLIC REFERENCES
> -----------------
> 
> This document lists the organizations that use Idpbuilder based on public information available in blog posts and videos.

Reviewer: ```suggestion
# Who uses idpbuilder?
---
As the CNOE Community grows, we'd like to keep track of our users. Please send a PR with your organization name if you are using idpbuilder CLI or source code.

```

If any organization would like to get added or removed please make a pull request by following these guidelines:

> * Please don't include your organization's logo or other trademarked material

Reviewer: ```suggestion
*  For the organization column make the organization name a link that points to the orgs website if available. Please don't include your organization's logo or other trademarked material.
```

> *  For the organization column make the organization name a link that points to the orgs website if available. Please don't include your organization's logo or other trademarked material.

Author: ```suggestion
* For the organization column make the organization name a link that points to the org website if available. Please don't include your organization's logo or other trademarked material.
```

Reviewer: this looks good apply the suggested changes

> * Add a reference (link to a public blog post, video, slides, etc) mentioning that Idpbuilder is used

Reviewer: ```suggestion
* Add a short description on how your organization is benefiting from using idpbuilder,  is encourage to also include link to a public (blog post, video, slides, git repo, etc.) that shows or mentions how Idpbuilder is used
```

There might organizations using it internally, and don't have a public url reference, and we want to be able to capture this adopters in this document.

> | Organization   | Reference                                                                           |

Reviewer: ```suggestion
| Organization   | Usage                                                                           |
```

> * Add a short description on how your organization is benefiting from using idpbuilder,  is encourage to also include link to a public (blog post, video, slides, git repo, etc.) that shows or mentions how Idpbuilder is used

Author: ```suggestion
* Add a short description on how your organization is benefiting from using idpbuilder. We encourage to also include a link to a public (blog post, video, slides, git repo, etc.) that shows or mentions how Idpbuilder is used
```

Reviewer: this looks good apply the suggested changes

|----------------|-------------------------------------------------------------------------------------|

> | New-York Times | <https://thenewstack.io/building-an-idp-with-help-from-the-open-source-cnoe-framework/>                                   |

Reviewer: ```suggestion
| [The New York Times](https://www.nytimes.com) | <https://thenewstack.io/building-an-idp-with-help-from-the-open-source-cnoe-framework/>                                   |
```
